scripts/install_deps_macOS.sh

```diff
@@ -1 +1 @@
-brew install boost glib llvm antlr antlr4-cpp-runtime pkgconfig rocksdb cmake boost-python3 qt5
+brew install boost glib antlr antlr4-cpp-runtime pkgconfig rocksdb cmake libwebsocketpp-dev nlohmann-json
```

Comment:
Homebrew doesn't have `-dev` packages, headers are included inside the main package.
```suggestion
brew install boost glib antlr antlr4-cpp-runtime pkgconfig rocksdb cmake libwebsocketpp nlohmann-json
```